var/spack/repos/builtin/packages/htslib/package.py

```diff
@@ -76,13 +81,23 @@ def url_for_version(self, version):
             url = "https://github.com/samtools/htslib/releases/download/{0}/htslib-{0}.tar.bz2"
             return url.format(version.dotted)
 
+    def flag_handler(self, name, flags):
+        if name == "cflags" and self.spec.satisfies("+pic"):
+            flags.append("-fPIC")
```

Comment:
Should use `cc_pic_flag`

Reply:
I was unaware of that! I'll look it up. Fantastic.